templates/test.html

```diff
@@ -1,5 +1,26 @@
 {% extends 'base.html' %}
 
 {% block content %}
-    <h1>supports page</h1>
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-12">
+                <div class="card">
+                    <h1 class="card-title">Accessibility Statement</h1>
+                    <div class="card-body">
+                        <p class="card-text">Over The Rainbow, (and associated such as OTR), is committed to facilitating the accessibility and usability of its Web site, https://www.overtherainbow.com,
+                            for all people with disabilities. Over The Rainbow has implemented and will continue to implement the relevant portions
+                            of the World Wide Web Consortium’s Web Content Accessibility Guidelines 2.1 Level AA (WCAG 2.1 AA) or higher as its web accessibility standard.</p>
+                        <p class="card-text">Please be aware that our efforts are ongoing. If at any time you have specific questions or concerns about the accessibility
+                            of any web page on https://www.overtherainbow.com, please contact us at info@overtherainbow.com. If you do encounter
+                            an accessibility issue, please be sure to specify the web page in your email, and we will make all reasonable efforts to make that page accessible for you.</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-sm-12 text-center brand-img">
+                <img class="img-fluid" src="/static/images/Logo1-tinypng.png" alt="OTR Logo">
+            </div>
+        </div>
+    </div>
```

Comment:
Also, the name of this file is `test.html`, and this needs to be updated to reflect the actual page it serves. Can the filename be ammended to `accessibility.html`, and then the appropriate view function in the app.py be updated to reflect this change?